src/log.ts

```diff
@@ -7,6 +7,9 @@ export const log = {
       process.stderr.write(`DEBUG: ${format(...args)}\n`);
     }
   },
+  warning: (...args: unknown[]) => {
+    process.stderr.write(`WARNING: ${format(...args)}\n`);
```

Comment:
should it be `process.stdout` or `process.stderr`, I noticed other helpers like `info` and `debug` also directed to `stderr`

Reply:
It must not be mixed with json and yaml output, therefore definitely STDERR.

Reply:
Added example with separate outputs from STDOUT and STDERR to demonstrate that the warning is in different stream.